README.md

```diff
@@ -13,46 +13,48 @@ access Microsoft Cloud service resources. This version of the connector only sup
 information of a specific account (currently logged-in user).
  
 For more information, go to the module(s).
-- [ballerinax/microsoft.teams](https://docs.central.ballerina.io/ballerinax/microsoft.teams/0.1.0)
+- [microsoft.teams](teams/Module.md)
  
-## Building from the Source
-### Setting Up the Prerequisites
-1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
- 
-  * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
- 
-  * [OpenJDK](https://adoptopenjdk.net/)
- 
-       > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed
-       JDK.
+## Building from the source
+### Setting up the prerequisites
+1.  Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
+   > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed
```

Comment:
This should be a step. Please refer to other connectors. 
If we need to have this as a note, please use markdown syntax.

Reply:
This is added as per the [common format agreed by the team](https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit#heading=h.96g5nsmszl71). The changes which will alter that format will be added in next release cycles